pkg/scheduler/controller/shoot/scheduler_control.go

```diff
@@ -156,7 +156,6 @@ func (c *defaultControl) ScheduleShoot(ctx context.Context, obj *gardencorev1bet
 	}
 
 	schedulerLogger.Infof("Shoot '%s' (Cloud Profile '%s', Region '%s') successfully scheduled to seed '%s' using SeedDeterminationStrategy '%s'", shoot.Name, shoot.Spec.CloudProfileName, shoot.Spec.Region, seed.Name, c.config.Schedulers.Shoot.Strategy)
-	c.reportSuccessfulScheduling(shoot, seed.Name)
```

Comment:
Why did you remove this event? I think it makes sense to keep it, just like the `Scheduled` event from the `kube-scheduler`.

Reply:
I removed it as we are limited in the amount of Shoot events we can persist. I think this is a difference to the kube-scheduler which assumes that events are not kept for weeks in etcd.
Initially, in response to a comment by @rfranzke we decided that we remove events signalling results (like scheduled successful or failed). Then we decided that we do still need errors as user feedback in the form of events.
Regarding `SuccessfulScheduling: is this an important user feedback that should be kept for weeks? How often does this event get fired? Depending on this information I would keep or delete it.

Reply:
This event gets only fired once after initial creation at the moment. I have no issues with keeping it, but I'm also fine with not keeping it ;-) Your call..

Reply:
As it is analogous to the kubescheduler and only fired once at creation I added the event again.